File: src/main/java/com/nndmove/app/config/ApplicationProperties.java
package com.nndmove.app.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * Properties specific to JHipster.
 * <p>
 * Properties are configured in the {@code application.yml} file.
 * See {@link tech.jhipster.config.JHipsterProperties} for a good example.
 */
@Getter
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
public class ApplicationProperties {

    private final Liquibase liquibase = new Liquibase();

    // jhipster-needle-application-properties-property

    // jhipster-needle-application-properties-property-getter

    @Setter
    @Getter
    public static class Liquibase {

        private Boolean asyncStart;
    }
    // jhipster-needle-application-properties-property-class
}

File: src/main/java/com/nndmove/app/config/ApplicationProperties.kt
package com.nndmove.app.config

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * Properties specific to JHipster.
 *
 *
 * Properties are configured in the `application.yml` file.
 * See [tech.jhipster.config.JHipsterProperties] for a good example.
 */
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
class ApplicationProperties {
    val liquibase: Liquibase = Liquibase()

    // jhipster-needle-application-properties-property
    // jhipster-needle-application-properties-property-getter
    class Liquibase {
        var asyncStart: Boolean? = null
    } // jhipster-needle-application-properties-property-class
}
